projects/robots/mobsya/thymio/controllers/thymio2_aseba/Thymio2AsebaHub.cpp

```diff
@@ -76,8 +76,8 @@ void Thymio2AsebaHub::connectionCreated(Dashel::Stream *stream) {
 }
 
 void Thymio2AsebaHub::incomingData(Dashel::Stream *stream) {
-  uint16 temp;
-  uint16 len;
+  unsigned short int temp;
```

Comment:
Was it possible to include `#include <stdint.h>`?

Reply:
Yes, but `stdint` uses `uint16_t` and not `uint16`, so I would anyhow have to rename all those types... Also `short int` was used at other places in the code, so, it was more homogeneous like that.